cwms-data-api/src/main/java/cwms/cda/data/dao/LocationLevelsDaoImpl.java

```diff
@@ -641,7 +641,7 @@ private static TimeSeries buildTimeSeries(ILocationLevelRef levelRef, Interval i
         TimeSeries timeSeries = new TimeSeries(null, size, size, timeSeriesId,
                 officeId, firstValueTime, lastValueTime, levelUnits,
                 java.time.Duration.ofSeconds(interval.getSeconds()),
-                null, null, locationTimeZone.getId());
+                null, null, locationTimeZone.getId(), null, null);
```

Comment:
I'm not a fan of null enums. I think UNVERSOINED is appropriate here

Reply:
done